plugins/modules/azure_rm_postgresqlflexibleserver.py

```diff
@@ -683,6 +683,9 @@ def __init__(self):
         self.results = dict(changed=False)
         self.state = None
 
+        self.managed_identity = {"identity": PostgreSQLFlexibleModels.UserAssignedIdentity,
```

Comment:
@p3ck This definition cannot be used here, because it will cause the following error, which can be detected by using 'ansible-test sanity plugin/modules/azure_rm_postgresqlflexibleserver.py

```e.g.
ERROR: plugins/modules/azure_rm_postgresqlflexibleserver.py:0:0: import-error: Exception attempting to import module for argument_spec introspection, 'name 'PostgreSQLFlexibleModels' is not defined'